modules/meta.analysis/DESCRIPTION

```diff
@@ -34,7 +34,8 @@ Imports:
     PEcAn.logger,
     MASS,
     PEcAn.settings,
-    rjags
+    rjags,
+    ggplot2
```

Comment:
If `ggplot2` is only used for optional arguments (that `gg=TRUE` argument) then it should probably be in `Suggests` rather than `Imports`.